packages/platform/enums/README.md

```diff
@@ -0,0 +1 @@
+Contains enums shared by platform api and packages
```

Comment:
I would add comment explaining that if V2 import from here it needs to import from platform-enums and other packages need to import from monorepo.ts